CONTRIBUTING.rst

```diff
@@ -77,13 +54,12 @@ Ready to contribute? Here's how to set up `pyfar` for local development.
    Now you can make your changes locally.
 
 5. When you're done making changes, check that your changes pass flake8 and the
-   tests, including testing other Python versions with tox::
+   tests::
 
     $ flake8 pyfar tests
     $ pytest
-    $ tox
 
-   To get flake8 and tox, pip install them into your virtualenv. The flake8 test must pass without any warnings for `./pyfar` and `./tests` using the default or a stricter configuration. Flake8 ignores `E123/E133, E226` and `E241/E242` by default. If necessary adjust the your flake8 and linting configuration in your IDE accordingly.
+   flake8 test must pass without any warnings for `./pyfar` and `./tests` using the default or a stricter configuration. Flake8 ignores `E123/E133, E226` and `E241/E242` by default. If necessary adjust the your flake8 and linting configuration in your IDE accordingly.
 
 6. Commit your changes and push your branch to GitHub::
```

Comment:
Since these change the UI from time to time and new tools might pop up, I would prefer the nerdy command-line-only documentation. That won't change much in the future and will keep maintaining the docs simple for us...

Reply:
Good point, what about adding at short hint to UIs in general, so it's clear for a newcomer that the command line is not the only possibility?

Reply:
Yes - that's the way! Do you want to add a line and also mention the Git command bash for Windows?